ocrd/ocrd/cli/ocrd_tool.py

```diff
@@ -119,12 +121,14 @@ def ocrd_tool_tool_dump(ctx):
 
 @ocrd_tool_tool.command('parse-params')
 @parameter_option
+@parameter_override_option
 @click.option('-j', '--json', help='Output JSON instead of shell variables', is_flag=True, default=False)
 @pass_ocrd_tool
-def ocrd_tool_tool_parse_params(ctx, parameter, json):
+def ocrd_tool_tool_parse_params(ctx, parameter, parameter_override, json):
     """
     Parse parameters with fallback to defaults and output as shell-eval'able assignments to params var.
     """
+    set_json_key_value_overrides(parameter, *parameter_override)
```

Comment:
I guess here also something must still be done to adapt for multiple `-p` (list-valued `parameter`), right?

Reply:
Yes, I'll merge this into https://github.com/OCR-D/core/pull/514 now to get both repeatable parameters and parameter overrides in v2.11.0

Reply:
Actually, that is handled in the click callback already, `kwargs["parameter"]` is already merged (in #514, that is).

Reply:
Oh, right, already forgot about that!